content/concepts/consensus-network/amendments/known-amendments.md

```diff
@@ -113,7 +149,34 @@ The sender or the receiver can cancel a Check at any time before it is cashed. A
 Introduces three new transaction types: CheckCreate, CheckCancel, and CheckCash, and a new ledger object type, Check. Adds a new transaction result code, `tecEXPIRED`, which occurs when trying to create a Check whose expiration time is in the past.
 
 
-## CryptoConditions
+### Clawback
+[Clawback]: #clawback
+
+| Amendment     | Clawback |
+|:--------------|:---------|
+| Amendment ID  | 56B241D7A43D40354D02A9DC4C8DF5C7A1F930D92A9035C4E12291B3CA3E1C2B |
+| Status        | In Dev   |
+| Default Vote (Latest stable release) | Yes |
+| Pre-amendment functionality retired? | No  |
+
+For regulatory purposes, some issuers must have the ability to recover issued tokens after they are distributed to accounts. For example, if an issuer were to discover that tokens were sent to an account sanctioned for illegal activity, the issuer could recover, or _claw back_ the funds.
+
+Clawback is disabled by default. To use clawback, you must set the `lsfAllowTrustLineClawback` flag using an `AccountSet` transaction.
+
+See [Clawback](https://opensource.ripple.com/docs/clawback/clawback-of-issued-currency/) for details on this amendment.
+
+
+### Cross Chain Bridges
+[CrossChainBridges]: #cross-chain-bridges
+
+| Amendment    | Cross Chain Bridges |
+|:-------------|:-----------------|
+| Amendment ID | N/A |
```

Comment:
I think "N/A" will trip up AmendmentBot. Also, it's probably better to put the Amendment ID here so you can identify it in the appropriate chains' ledger data, even if the final amendment may have a different name and ID.

I believe the current amendment ID for XChainBridge is this:

```suggestion
| Amendment ID | C98D98EE9616ACD36E81FDEB8D41D349BF5F1B41DD64A0ABC1FE9AA5EA267E9C |
```